oap-server/server-alarm-plugin/src/test/java/org/apache/skywalking/oap/server/core/alarm/provider/WebhookCallbackTest.java

```diff
@@ -58,7 +58,10 @@ public void stop() throws Exception {
     public void testWebhook() {
         List<String> remoteEndpoints = new ArrayList<>();
         remoteEndpoints.add("http://127.0.0.1:8778/webhook/receiveAlarm");
```

Comment:
This is potential ci failure reason, right? Do you have any suggestion to improve this?